program/image-classification-tflite/classification.cpp

```diff
@@ -59,6 +59,8 @@ int main(int argc, char* argv[]) {
         throw string("Failed to construct interpreter");
       if (interpreter->AllocateTensors() != kTfLiteOk)
         throw string("Failed to allocate tensors");
+
+      //interpreter->SetNumThreads(settings.number_of_threads());
```

Comment:
Hmm, why is it added but commented out? :)

Reply:
Fixed